corehq/apps/ota/tests/test_app_aware_restore.py

```diff
@@ -134,7 +134,7 @@ def test_report_fixtures_provider_with_app_that_doesnt_have_reports(self):
         with patch.object(ConfigurableReportDataSource, 'get_data') as get_data_mock:
             get_data_mock.return_value = self.rows
             fixtures = call_fixture_generator(report_fixture_generator, self.user, app=self.app3)
-        self.assertEqual(len(fixtures), 0)
+        self.assertEqual(len(fixtures), 1)
```

Comment:
How come this is now 1?

Reply:
Whenever we don't have a v1 report we send down an empty fixture. This is to purge the old v1 fixtures from phone

https://github.com/dimagi/commcare-hq/pull/18273/files/314fe154ff8bf1e2f15184f5911054e45e89ad17#diff-b00a086d998daf07fcf62372491b3ff3R157